tests/component.js

```diff
@@ -208,6 +208,34 @@ describe( 'CKEditor Component', () => {
 		} );
 	} );
 
+	describe( 'when watched property changed before editor initialized', () => {
```

Comment:
Do you think it would make sense to test it in more integration manner? So checking how the entire initialization flow behaves when watched props are changed before editor is initialized and also see if the values are correct after editor is initialized.

Reply:
It results in very strange tests like this one: https://github.com/ckeditor/ckeditor4-vue/blob/b885443b01d013fcd088fbc6509add827db60505/tests/component.js#L238-L291

So I decided to go an easy way. But I will take a look if I can find out some "not too complicated" integration test.

Reply:
It seems like writing a proper integration test for it is a bit of cumbersome, and as this test already validates correct behavior, let's skip it.

Reply:
Makes sense for now. If we will need this in the future we need to thing about more generic and more elegant approach.